components/radioButtons.tsx

```diff
@@ -1,12 +1,28 @@
 import { Root, Item, Indicator } from '@radix-ui/react-radio-group'
+import type {
+  RadioGroupProps,
+  RadioGroupItemProps,
+  RadioIndicatorProps,
+} from '@radix-ui/react-radio-group'
 
-export const RadioButtons = (props: any) => (
-  <Root {...props} className={`${props.className}`} />
-)
+export interface RadioButtonsProps extends RadioGroupProps {
+  variant: 'radio-button-bar' | '' | undefined
```

Comment:
dont pass in a shortcut—shortcuts are utility classes that can be applied to other components. if there are other formats planned for this component i suggest one of two things:
- change the first literal in this type to `’bar’`
- use separate components, calling this one RadioButtonsBar

Reply:
is there a different word besides variant that wouldn’t conflict with terminology in windicss? maybe layout or format

Reply:
Changed to "treatment" and changed the title to "button-bar", which could see use other places.

Reply:
Changed the prop name to "treatment"